runtime/libffi/z/ffi64.c

```diff
@@ -875,58 +921,64 @@ ffi_prep_cif_machdep(ffi_cif *cif)
 	{
 		type = ffi_check_struct_type (*ptr);
 
+      (*ptr)->type = ffi_check_struct_for_complex(*ptr);
+
+
```

Comment:
One blank line at a time is sufficient.

Reply:
Done in https://github.com/eclipse-openj9/openj9/compare/6322f343e9e88af4b64f7d08586a47a7ecb89b32..583867825be4477a8319f21ffb952780631fbeb3